### PR TITLE
Do not check to see if term exists

### PR DESCRIPTION
**Pull request by spacedmonkey**
Trac ticket: https://core.trac.wordpress.org/ticket/57966
---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

File: src/wp-includes/post.php [hunk 1]
```diff
@@ -7660,36 +7658,14 @@ function wp_queue_posts_for_term_meta_lazyload( $posts ) {
...
+							$term_ids[] = (int) $term_id;
 						} elseif ( isset( $term_id->term_id ) ) {
-							$prime_taxonomy_ids[ $taxonomy ][] = (int) $term_id->term_id;
-							$prime_term_ids[]                  = (int) $term_id->term_id;
+							$term_ids[] = (int) $term_id->term_id;
 						}
 					}
 				}
 			}
 		}
```
Comment (mukeshpanchal27) [inline src/wp-includes/post.php:7671]:
```suggestion
		}
		$term_ids = array_unique( $term_ids );
```
Remove duplicate teem ids from the array.

**Review by mukeshpanchal27 — COMMENTED** [review]
Thanks @spacedmonkey, Overall look good to me.

- Use unique term ids as there may be duplicate term ids
- It's good to add `@since` annotation with message that "As performance improvement remove `get_term` call.

**Comment by spacedmonkey** [comment]
> Thanks @spacedmonkey, Overall look good to me.
> 
> * Use unique term ids as there may be duplicate term ids
> * It's good to add `@since` annotation with message that "As performance improvement remove `get_term` call.

I am not sure about adding that as a comment in code. That is what commit messages are for.

**Review by joemcgill — APPROVED** [review]
I think approach makes sense based on the prior conversation in https://core.trac.wordpress.org/ticket/57966.

**Review by mukeshpanchal27 — APPROVED** [review]
Thanks @spacedmonkey and @joemcgill for clarification.

**Comment by spacedmonkey** [comment]
Committed https://github.com/WordPress/wordpress-develop/commit/9ef194bc4b9bfaf359f1208dfe927b6688c299f0

**Pull request closed without merging**